### ISSUE TITLE
Set charging amps with MQTT

### ISSUE THREAD

**Issue by h4ckst0ck** [bassmaster187/TeslaLogger#740]
It would be very helpful to be able to set the charging current via MQTT and then have it set in the vehicle via the Tesla API. This would make it much easier to use surplus electricity from a photovoltaic system. I have teslalogger connected via mqtt to openhab, and there accordingly the tesla binding not used. I miss with the solution also nothing - except the setting of the charging current.

**Comment by bassmaster187** [comment]
Does a HTTP Request solve your problem as well?

**Comment by h4ckst0ck** [comment]
Yes this would be ok, too. This is probably the easier solution for other users as well.

**Comment by Adminius** [comment]
Example: http://raspberry:5000/command/1/set_charging_amps?16


@h4ckst0ck please close this issue if it solved
